.github/workflows/qemu.yml

```diff
@@ -59,54 +61,46 @@ jobs:
         chpasswd:
           list: |
             root:password
-            cxltest:password
+            testuser:password
           expire: False
         EOF
 
         cat > meta-data << EOF
-        instance-id: cxl-test
-        local-hostname: cxl-test
+        instance-id: qemu-test
+        local-hostname: qemu-test
         EOF
 
         sudo -Sk genisoimage -output ubuntu-cloud-init.iso -volid cidata -joliet -rock ./user-data ./meta-data
+
     - name: Download ubuntu image
       run: wget https://cloud-images.ubuntu.com/releases/lunar/release/ubuntu-23.04-server-cloudimg-amd64.img
+
     - name: Resize image
       run: qemu-img resize ./ubuntu-23.04-server-cloudimg-amd64.img +4G
-    - name: Build
-      run: |
-        scripts/qemu/start_qemu.sh scripts/qemu/configs/default.xml
 
-        if [ ${{ github.event_name }} = 'pull_request' ]; then
-            CI_REPO="${{ github.event.pull_request.head.repo.full_name }}"
-        else
-            CI_REPO="$GITHUB_REPOSITORY"
-        fi
+    - name: Build UMF in QEMU
+      run: |
+        umf/scripts/qemu/start_qemu.sh default.xml
 
-        scp -P 2222 ${{github.workspace}}/scripts/qemu/run-build.sh cxltest@127.0.0.1:/home/cxltest
-        scp -P 2222 ${{github.workspace}}/scripts/qemu/run-tests.sh cxltest@127.0.0.1:/home/cxltest
-        ssh cxltest@127.0.0.1 -p 2222 -t "bash /home/cxltest/run-build.sh https://github.com/$CI_REPO ${{env.CI_BRANCH}}"
+        rsync -az -e "ssh -p 2222" ${{github.workspace}}/umf/ testuser@127.0.0.1:/home/testuser/
```

Comment:
Could you add a comment what you copy here (using `rsync`)?

Reply:
Done.